src/components/Card/CardToolbar.jsx

```diff
@@ -85,9 +88,10 @@ const CardToolbar = ({
   availableActions,
   timeRange,
   onCardAction,
+  className,
 }) => {
   return isEditable ? (
-    <div className="card--toolbar">
+    <div className={classNames(className, 'card--toolbar')}>
```

Comment:
Maybe go ahead and add the prefix while you're here?

Reply:
see comment below